Разметка всех страниц

BondarenkoDenis opened the pull request:
сделал разметку пока что только первой страницы. каталог с гостиницами пока не делал, тк есть большой вопрос с классами и их именованием. и вообще какой то винегрет вышел.

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2229813/tasks/5)

BondarenkoDenis committed:
разметка страницы

разметка страницы

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+        <p class="about-text">Сделано в <a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a></p>
+    </footer>
 
     <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
```
Удалить этот абзац

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Гостиницы</a>
+          </li>
+          <ul class="navigation-list navigation-user">
```
Здесь лучше использовать другие классы, т.к. по внешнему виду имеет мало общего с основным меню
navigation-user-list
navigation-user-item 
и т.д.

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+    <main class="main-index">
+
+      <section class="hero">
+        <img class="hero_logo" src="#" alt="Добро пожаловать в Седона">
```
Через тире все классы задавай - hero-logo

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+заслуживающий большего!</h1>
+      <p>Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+      </section>
+      <section class="advantage1">
```
Сделай просто advantage без единичек

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+        <ul class="advantage-list">
+          <li class="advantage-item">
+            <h3 class="advantage-title">Настоящий городок</h3>
+            <img class="advantage-line" src="#" alt="разделитель">
```
Разделители это декоративный элемент, соответственно не нужно размечать его в разметке

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+        <img class="hero_logo" src="#" alt="Добро пожаловать в Седона">
+      <h1>Седона — небольшой городок в Аризоне,
+заслуживающий большего!</h1>
+      <p>Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
```
Перенести заголовок и абзац в секцию advantage и задать классы advantage-title и advantage-text

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+            <p class="advantage-text">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+          </li>
+            <li class="advantage-item">
+              <img class="advantage-image" src="#" alt="отель в Седона">
```
Картинка должна быть внутри первого пункта li

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+              <li class="advantage-item">
+                <h3 class="advantage-title">Небольшая площадь</h3>
+                <img class="advantage-line" src="#" alt="разделитель">
+                <p class="advantage-text">Все достопримечательности находятся очень близко</p>
```
Сделай классы лучше advantage-item-title и advantage-item-text, чтобы не пересекалось с заголовком и абзацем секции

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+      <section class="advantage2">
+        <h2>Приезжайте в Седону отдохнуть в комфорте и уюте!</h2>
+        <p>Опытный персонал и качественное обслуживание!</p>
+      </section>
```
секцию можно назвать feature, внутри заголовок и абзац назвать используя часть feature-*

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+            <img class="souvenir" src="#" alt="сувениры">
+            <h3 class="eat-food-title">сувениры</h3>
+            <p class="eat-food-text">Не только китайского, но и настоящего местного производства!</p>
+          </li>
```
Список и секция у тебя не закрыт, в любом случае список нужно перенести в feature и поменять классы (пример с секцией advantage), а секцию удалить

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+            <p class="eat-food-text">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+          </li>
+          <li class="eat-food-item">
+            <img class="eat" src="#" alt="еда">
```
Эти картинки не особо тянут на контентные, можно будет разместить их через стили, добавив дополнительный уникальный класс на li

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+          <section class="hostel-search">
+            <h2>Заинтересовались?</h2>
+            <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+            <a class="button-hostel-search" href="#">поиск гостиницы в Седоне</a>
```
Для всех почти одинаково выглядящих кнопок/ссылок можно добавить класс button (будет основной класс и содержать преобладающий в макете цвет кнопки), дополнительный цветы можно добавить через дополнительный класс типа button-red, а так же добавить дополнительной класс от секции, в данном случае:
hostel-search-button

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+
+          <section class="hostel-search">
+            <h2>Заинтересовались?</h2>
+            <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
```
Добавить классы для заголовка и абзаца от секции

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+          </section>
+
+          <section class="mailing-list">
+            <h3 class="mailing-list-text">Подпишитесь на рассылку</h3>
```
поменять тег на h2 и класс на mailing-list-title

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+          <section class="mailing-list">
+            <h3 class="mailing-list-text">Подпишитесь на рассылку</h3>
+            <p class="mailing-list-text">Только полезная информация и никакого спама, честное бойскаутское!</p>
+            <form class="appointment-form" action="https://echo.htmlacademy.ru/" method="post">
```
поменять класс mailing-list-form

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+        <h2 class="page-footer-title">соцсети</h2>
+        <ul class="footer-social-list">
+            <li class="footer-social-item">
+              <a class="button" href="#">
```
Поменять у ссылках соц. сетях класс на   footer-social-link
Ссылки можно сразу проставить по требованию в макете

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+
+       <section class="footer-contacts">
+        <address class="footer-contacts-phone">
+          <a class="footer-contacts-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
```
footer-contacts-phone-link

Stalyer commented on index.html:
```diff
@@ -3,8 +3,157 @@
+        </address>
+        </section>
+
+        <p class="about-text">Сделано в <a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a></p>
```
copyright и copyright-link
Внутри ссылки будет картинка

BondarenkoDenis committed:
разметка всех страниц проекта

сделал разметку всех страниц проекта. главной учел все замечания (по существу все сделал заново)
вышло много строк - делал специально для лучшего понимания.
там, где сомнения - оставил комментарии

BondarenkoDenis committed:
список сортировки в каталоге

список сортировки в каталоге
дописать классы в сортировке
